caddyhttp/httpserver/context.go

```diff
@@ -57,6 +58,24 @@ func (c Context) Header(name string) string {
 	return c.Req.Header.Get(name)
 }
 
+// Env gets a map of the environment variables.
+func (c Context) Env() map[string]string {
```

Comment:
Let's just leave it as-is for now, we can optimize later. This function is good for now.